.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Bug report: Netdata Cloud
```

I think I broke your .yml syntax when I recommended adding in the colon `:`. My bad! I think some quotes will fix it, though.

```suggestion
  - name: "Bug report: Netdata Cloud"
```